libvcx/src/aries/handlers/proof_presentation/verifier/verifier.rs

```diff
@@ -33,6 +34,12 @@ impl Verifier {
         })
     }
 
+    pub fn from_presentation_request(source_id: &str, name: &str, presentation_request: &PresentationRequest) -> VcxResult<Verifier> {
```

Comment:
Nice :-) Soon Rust API will be twice as richer as C API.

Reply:
Good that you commented on this, because it made me realize I ended up not using this constructor. It can be useful, so I guess it can stay as long as I write a unit test for it, but on the other hand, YAGNI argument applies as well... What do you think?

Reply:
I agree if you don't need this, let's delete

Reply:
Agreed